TTS/tts/configs/delightful_tts_config.py

```diff
@@ -0,0 +1,118 @@
+from dataclasses import asdict, dataclass, field
+from typing import Dict, List
+
+from TTS.tts.configs.shared_configs import BaseTTSConfig
+from TTS.tts.models.delightful_tts import DelightfulTtsArgs, DelightfulTtsAudioConfig, VocoderConfig
+
+
+@dataclass
+class DelightfulTTSConfig(BaseTTSConfig):
+
```

Comment:
You can consider typing docstrings for the config arguments. I'd help you understand architecture better.